### PR TITLE
Upgrade to Tailwind 3

### PR DESCRIPTION
**Pull request by abstractalgo**
https://tailwindcss.com/docs/upgrade-guide

### CODE REVIEW

**Comment by jaril** [comment]
> 

This one's unrelated to this change so feel free to ignore. I'll fix it in a separate thread.

**Comment by jaril** [comment]
The QA Wolf test failure is unrelated to this change so feel free to ignore. I'll fix it in a separate thread.

File: tailwind.config.js [hunk 1]
```diff
@@ -1,12 +1,6 @@
 module.exports = {
-  purge: {
-    content: ["./src/**/*.{js,jsx,ts,tsx}"],
-  },
-  darkMode: false, // or 'media' or 'class'
+  content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    screens: {
-      short: { raw: "(max-height: 615px)" },
-    },
```
Comment (jaril) [inline tailwind.config.js:9]: This is being used in the upload screen. What's the reason for taking this out?

File: tailwind.config.js [hunk 2]
```diff
@@ -33,12 +27,5 @@ module.exports = {
       },
     },
   },
-  variants: {
-    width: ["hover", "group-hover"],
-    extend: {
-      borderWidth: ["last"],
-      textColor: ["disabled"],
-    },
-  },
```
[the remark this answers is not in the document]
Comment (abstractalgo) [reply tailwind.config.js:42]: Yep, per https://tailwindcss.com/docs/upgrade-guide#remove-variant-configuration

File: tailwind.config.js (hunk 1 above)
Comment (abstractalgo) [reply tailwind.config.js:9]: That search-fu has failed me to find reference to it. Will revert that part.

**Pull request merged**